test/test_helper.rb

```diff
@@ -10,14 +10,19 @@
 
 WebMock.allow_net_connect!
 
-Minitest::Reporters.use! Minitest::Reporters::DefaultReporter.new
-# Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
-# Minitest::Reporters.use! Minitest::Reporters::ProgressReporter.new
+Minitest::Reporters.use! [Minitest::Reporters::DefaultReporter.new(color: true)]
+
+unless Webpacker.compiler.fresh?
+  puts "== Webpack compiling =="
+  Webpacker.compiler.compile
```

Comment:
Doing this was dramatically faster. I know we tell Webpack to compile in the test env but doing it at the beginning took the test suite from minutes to 12 seconds locally.